Snowdog_Components/components/Templates/catalog/grid/_grid-variables.scss

```diff
@@ -0,0 +1,8 @@
+// Catalog Grid
+$catalog-grid__gap                      : $spacer--medium !default;
+$catalog-grid__margin                   : 0 !default;
+$catalog-grid__padding                  : $spacer--medium 0 $spacer--large 0 !default;
+$catalog-grid__column-base-fraction     : 1fr !default;
+$catalog-grid__template-columns         : $catalog-grid__column-base-fraction !default;
+$catalog-grid__template-columns\@small  : $catalog-grid__column-base-fraction $catalog-grid__column-base-fraction !default;
+$catalog-grid__template-columns\@medium : $catalog-grid__column-base-fraction $catalog-grid__column-base-fraction $catalog-grid__column-base-fraction !default;
```

Comment:
I would consider changing the syntax to repeat() https://www.digitalocean.com/community/tutorials/css-css-grid-layout-repeat-notation. I find it more readable. what do you think?

Reply:
Good idea, fixed